packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -474,6 +492,14 @@ MeetingUtil.startInternalStats = (meeting) => {
           history: true,
           onEvent: (event) => {
             if (event.kind === 'video' && event.stat === 'bytesSent') {
+              Trigger.trigger(
+                this,
+                {
+                  file: 'meeting/util',
+                  function: 'video'
+                },
+                EVENT_TRIGGERS.MEETING_LOCAL_MEDIA_STOPPED,
```

Comment:
Could you add a payload for the type of media? Also, the function scope for all of these should be the calling function `startInternalStats`

```js
              Trigger.trigger(
                this,
                {
                  file: 'meeting/util',
                  function: 'startInternalStats'
                },
                EVENT_TRIGGERS.MEETING_LOCAL_MEDIA_STOPPED,
                {
                  type: 'video'
                }, 
```